hms-app/src/components/HeaderMenu.tsx

```diff
@@ -116,6 +116,9 @@ export default function HeaderMenu({ links }: HeaderSearchProps) {
       <Container>
         <div className={classes.inner}>
           <h1>HMS</h1>
+          <Button onClick={() => localStorage.clear()}>
```

Comment:
is it for testing? you could do this through the dev console by clearing out the localstorage in the dev console